flashcards/backend/users/management/commands/create_test_data.py

```diff
@@ -4,16 +4,17 @@
 from django.db import transaction
 
 from django.conf import settings
```

Comment:
[imports by PEP8](https://peps.python.org/pep-0008/#imports)
Please fix ordering of the imports, empty line in line 5? Shouldnt' it be after curremt line 6?

Reply:
ok, now it's updated